client/js/screens/SingleSelfGuided/SingleSelfGuidedContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+import SingleSelfGuided from './SingleSelfGuided';
+import PropTypes from 'prop-types';
+import {Query} from 'react-apollo';
+import {QUERY_SELFGUIDED_TOUR} from '../../apollo/queries';
+import Loader from '../../components/Loader';
+import {Text, TouchableOpacity, Image} from 'react-native';
+
+class SingleSelfGuidedContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      mockId: 'ck435ktpmunrv0b00xbq37xme',
```

Comment:
what is this id for? Can you use dynamic data?

Reply:
Removed